internal/driver/interactive.go

```diff
@@ -70,6 +70,11 @@ func interactive(p *profile.Profile, o *plugin.Options) error {
 					value = strings.TrimSpace(value)
 				}
 				if v := pprofVariables[name]; v != nil {
+					// All non-bool options require inputs
+					if v.kind != boolKind && value == "" {
+						o.UI.PrintErr(fmt.Errorf("please input a value, e.g. %v", name+"=<val>"))
```

Comment:
Nit:
Here, I think %s would be more idiomatic than %v. 
The following would be standard for Golang:
```
o.UI.PrintErr(fmt.Errorf("please input a value, e.g. %s=<val>", name))
```

Reply:
Done.
Really appreciate those comments to help me write better Go code. Thanks!